### ISSUE TITLE
mono_trg using wrong vocab file

### ISSUE THREAD

**Issue by kpu** [mozilla/firefox-translations-training#90]
Looking at `train/fr-en/data/fr-en/fr-en-prod/translated/mono_trg/file.00.out` the quality is terrible.

```
chaleureux infrared heat dans  Deputy, in abuse will Par data sur est l of common to matière que map of in fire.
tiers the character cette quality- mariage to throw her legs open to her major C la curid des of When de être.
Knighte, front, pulls kids in a canoe through a flooded mobile home park in per City,UP., Aug.
r of silence hommeselling also appeared 3 in Dec.
```

I think this is supposed to be French.  @XapaJIaMnu took a look and thinks the wrong vocab.spm is being used, namely `train/fr-en/models/fr-en/fr-en-prod/vocab/vocab.spm` appears in file.00.log when it should be `data/models/fr-en/fr-en-prod/vocab/vocab.spm`

Possible it's a mess we made on our end.  Copying @ugermann and @XapaJIaMnu

**Comment by XapaJIaMnu** [comment]
I think it may have happened due to copying the files over to a new location..

**Comment by ugermann** [comment]
So here's what I think happened (or something along these lines).

- I copied over the backtanslation model and the vocab file from Nick's run for fr-en, because the last thing that was working
- did back-translation, checked quality (and I did)
- discovered issues with corpus cleaning 
- fixed those re-did the corpus cleaning
- that triggered a re-build of the vocabulary and should have triggered a re-training of the backward model
- but: when you re-run marian on a finished model, it just quits, so that model didn't get re-built

So the problem is that a re-build of a trained-to-completeion model actually doesn't train a new model but just skips that step.
The bad news is that fr->en actually did use the broken back-translations. En-fr didn't because I trained everything from scratch there and by that time had worked out most of the things that went wrong with fr->en.
